### ISSUE TITLE
Ethernet Delimiter is incorrectly spelled 'Dilimeter'

### ISSUE THREAD

**Issue by Amomum** [eldarkg/emdr1986x-std-per-lib#39]
In structure ETH_InitTypeDef there is a field - Dilimeter. It should be spelled 'D**e**limiter' (as it was spelled in older versions of SPL where everything was prefixed 'opora', as it is spelled in datasheet and in plain english).

I'm afraid that renaming the filed this will be breaking API change :(

**Comment by eldarkg** [comment]
@Amomum I don't that this is a big problem.
Can you create a pull request?

**Comment by Amomum** [comment]
@eldarkg I sure can but this structure is part of public API, so changing this field will break compilation. And it will break compatibility with 'official' SPL
I have no idea if this is a real problem though.

So I'm not sure if we should change it

**Comment by eldarkg** [comment]
@Amomum Early I applied [one](https://github.com/eldarkg/emdr1986x-std-per-lib/commit/146d36d04c28c4d0bedbc71477188387d18806c7) for ADC.
I think we can apply another one.
The official lib should be updated too (but the last from 06.10.2015 - it is so old. The current SPL fix some issues that official don't).
